packages/appstore/src/components/modals/real-wallets-upgrade/components/wallets-intro/wallets-intro.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { Text, Icon } from '@deriv/components';
+import { observer, useStore } from '@deriv/stores';
+import getWalletsIntroContent from 'Constants/wallets-intro-content-config';
+import './wallets-intro.scss';
+
+type TWalletsIntro = {
+    title: string;
+    description: string;
+    bullets: string[];
+    eu_user?: boolean;
+    image?: React.ReactNode;
+};
+
+type TWalletsIntroComponent = {
+    is_eu: boolean;
+    current_step: number;
+};
+
+const WalletsIntroComponent = observer(({ image, title, description, bullets }: TWalletsIntro) => {
+    const { ui } = useStore();
+    const { is_mobile } = ui;
+
+    const text_title_size = is_mobile ? 'xsm' : 'l';
+    const text_body_size = is_mobile ? 's' : 'm';
+    const text_info_size = is_mobile ? 'xs' : 's';
+    const form_line_height = is_mobile ? 'm' : 'l';
```

Comment:
can we separate these constants out into its own util files? cause I see that there are some other files using the exact same value checks like `is_mobile ? 's' : 'm'`

Reply:
Not sure it's a good idea, because every component has own design and we don't have any rules for this. Because some components can be with sizes `s/m`, some with `s/l`, some with `s/xl` and next... It's more easy to create sizes inside of components.